app/views/admin/claim-teacher-1.html

```diff
@@ -0,0 +1,178 @@
+
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  DfE Maths and Physics Service Prototype
+{% endblock %}
+
+{% block beforeContent %}
+  {% from "phase-banner/macro.njk" import govukPhaseBanner %}
+  {{ govukPhaseBanner({
+    tag: {
+      text: "alpha"
+    },
+    html: 'This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.'
+  }) }}
+
+  {% from "back-link/macro.njk" import govukBackLink %}
+  {{ govukBackLink({
+    text: "Back",
+    href: "claims"
+  }) }}
+
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">Retention payment claim details:</h1>
+
+      <form method="POST" action="confirmation">
+        {% from "summary-list/macro.njk" import govukSummaryList %}
+
+        <h2 class="govuk-heading-m">Teacher details</h2>
+        {{ govukSummaryList({
+          classes: 'govuk-!-margin-bottom-9',
+          rows: [
+            {
+              key: {
+                text: "Teacher name"
+              },
+              value: {
+                text: "Jane Jones"
+              }
+            },
+            {
+              key: {
+                text: "Teacher reference number (TRN)"
+              },
+              value: {
+                text: "2316748"
+              }
+            },
+            {
+              key: {
+                text: "School"
+              },
+              value: {
+                text: "Moorside High School"
+              }
+            }
+          ]
+        }) }}
+
+        <h2 class="govuk-heading-m">Claim details for Jane Jones</h2>
+        {% set adminQts = data['admin-qts'] if data['admin-qts'] else '<p class="answer-incomplete">Incomplete</p>' %}
```

Comment:
A bit of inconsistent casing here, the others are all camelCase. Perhaps call it `data['adminQts']`?